### PR TITLE
Replace removed endpoint for setting entitlements admin

### PR DESCRIPTION
**Pull request by dvelinovska**
Remove set entitlements admin from user-integration
Add set admins on accessgroup-integration
Replace the old endpoint with the new on in the configuration

### CODE REVIEW

File: src/main/java/com/backbase/ct/bbfuel/configurator/ServiceAgreementsConfigurator.java [hunk 1]
```diff
@@ -15,7 +15,11 @@
 import com.backbase.integration.accessgroup.rest.spec.v2.accessgroups.serviceagreements.Participant;
 import com.backbase.integration.accessgroup.rest.spec.v2.accessgroups.serviceagreements.ServiceAgreementPostResponseBody;
 import java.util.ArrayList;
+import java.util.Collections;
 import java.util.Set;
+
```
Comment (ButAds) [inline src/main/java/com/backbase/ct/bbfuel/configurator/ServiceAgreementsConfigurator.java]: Please use the backend coding style guide.

File: src/main/java/com/backbase/ct/bbfuel/client/accessgroup/ServiceAgreementsIntegrationRestClient.java [hunk 2]
```diff
@@ -18,18 +20,19 @@
...
     private static final String ENDPOINT_SERVICE_AGREEMENTS_BY_ID = ENDPOINT_SERVICE_AGREEMENTS + "/%s";
+    private static final String ADD_ADMINS_IN_SA = ENDPOINT_SERVICE_AGREEMENTS + "/admins/add";
 
     @PostConstruct
     public void init() {
         setBaseUri(config.getDbs().getAccessgroup());
         setVersion(SERVICE_VERSION);
     }
 
+
```
Comment (richard-blank) [inline src/main/java/com/backbase/ct/bbfuel/client/accessgroup/ServiceAgreementsIntegrationRestClient.java]: not needed

File: src/main/java/com/backbase/ct/bbfuel/client/user/UserIntegrationRestClient.java [hunk 3]
```diff
@@ -22,41 +20,20 @@
...
-                .statusCode(SC_OK);
-        }
-    }
-
     public void ingestUserAndLogResponse(UsersPostRequestBody user) {
         Response response = ingestUser(user);
 
-        if (response.statusCode() == SC_BAD_REQUEST &&
+        if (response.statusCode() == SC_BAD_REQUEST
+            &&
```
Comment (richard-blank) [inline src/main/java/com/backbase/ct/bbfuel/client/user/UserIntegrationRestClient.java]: i would not have this on a separate line

Comment (dvelinovska) [reply src/main/java/com/backbase/ct/bbfuel/client/user/UserIntegrationRestClient.java]: fixed

**Pull request merged**